cl/citations/tasks.py

```diff
@@ -163,12 +163,19 @@ def find_citations_and_parentheticals_for_opinion_by_pks(
 
             parentheticals = []
             for _opinion, _citations in citation_resolutions.items():
+                # Currently, eyecite has a bug where parallel citations are
+                # detected individually. We avoid creating duplicate parentheticals
+                # because of that by keeping track of what we've seen so far.
+                parenthetical_texts = set()
```

Comment:
This is a pretty good fix. I'd prefer if we could fix it upstream though in `eyecite`, and then fix it properly here, rather than hack around it. That'd make our code better, and would provide a nice enhancement to eyecite users to. See: https://github.com/freelawproject/eyecite/issues/76